config/locales/sv.yml

```diff
@@ -591,6 +597,11 @@ sv:
 
     index:
       title:  Hitta H-märkt företag
+      meta:
+        title: "Hitta H-märkt hundföretag, hundinstruktör"
+        description: "Hitta ett etiskt svenskt hundföretag som har tjänat H-märkt. Hitta en hundinstruktör, Hundbolaget, hundens entreprenör."
```

Comment:
description: "Här hittar du etiska svenskta H-märkta hundföretag. Du hittar bland annat hundinstruktörer, hundpsykologer, hunddagis, trim med mera."

Reply:
The description and title are _nearly_ the same as the defaults (`meta.default.title` and `meta.default.description`).  Can this view (companies index) just use the defaults instead?

Perhaps they "Hitta" is important.  In which case, should these entries be the defaults?